immix/src/allocator/big_obj_allocator.rs

```diff
@@ -101,7 +101,7 @@ impl BigObjAllocator {
     pub fn return_chunk(&mut self, obj: *mut BigObj) {
         let _lock = self.lock.lock();
         let size = unsafe { (*obj).size };
-        println!("ret_chunk: {:p}[size {}]", obj, size);
+        // println!("ret_chunk: {:p}[size {}]", obj, size);
```

Comment:
could add a todo: use debug logger to print logs hence we don't need to comment it.